railties/test/isolation/abstract_unit.rb

```diff
@@ -153,6 +153,7 @@ def build_app(options = {})
 
     def teardown_app
       ENV["RAILS_ENV"] = @prev_rails_env if @prev_rails_env
+      FileUtils.rm_rf(tmp_path)
```

Comment:
Why was this added?

Reply:
Intentionally, see description of commit [Fix error `Errno::ENOSPC: No space left on device - sendfile`](https://github.com/rails/rails/pull/30101/commits/d6aea9e087cbcf09f3a7fb487e25f873d76f24a5)